src/controllers/dissolved-search/search.controller.ts

```diff
@@ -77,7 +69,82 @@ const route = async (req: Request, res: Response) => {
             errorList: [companyNameErrorData],
             templateName: templatePaths.NO_RESULTS
         });
-    };
+    }
+
 };
 
+export const sanitiseCompanyName = (companyName) => {
+    const sanitisedName = escape(companyName);
+
+    return sanitisedName;
+}
+
+export const formatPostCode = (postCode) => {
+    let halfPostCode;
+    let trimmedPostCode;
+
+    if (postCode != null) {
+        const newPostCode = postCode.split(" "); 
+        halfPostCode = newPostCode[0];
+
+        trimmedPostCode = halfPostCode.slice(0, 4);
+    }
+
+    return trimmedPostCode;
+}
+
+export const formatDate = (unformattedDate) => {
```

Comment:
`formatDate`, `formatPostcode` and `sanitiseCompanyName`  could be moved into a separate service or utils class just to reduce the size of the controller. However its no biggie so just a suggestion 👍